modules/default/calendar/calendar.js

```diff
@@ -488,9 +503,11 @@ Module.register("calendar", {
 									timeWrapper.innerHTML = CalendarUtils.capFirst(this.translate("DAYAFTERTOMORROW"));
 								}
 							}
+							Log.info("event fullday");
 						} else if (event.startDate - now < this.config.getRelative * ONE_HOUR) {
+							Log.info("not full day but within getrelative size");
 							// If event is within getRelative hours, display 'in xxx' time format or moment.fromNow()
-							timeWrapper.innerHTML = CalendarUtils.capFirst(moment(event.startDate, "x").fromNow());
+							timeWrapper.innerHTML = `${CalendarUtils.capFirst(moment(event.startDate, "x").fromNow())}`; // C ${event.startDate};
```

Comment:
do we need the "C" stuff?

Reply:
also, why the string manipulation?

Reply:
sorry, the A/B/C were debugging info

string maniupulation???  where?? I had just added debug info to know which statement was firing.. removed the comments

Reply:
replied

Reply:
fixed